src/components/VideoListItem/VideoListItem.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { KeyboardEvent, memo, useState } from 'react';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import styles from './VideoListItem.module.scss';
+
+import type { ImageData } from '#types/playlist';
+import Image from '#src/components/Image/Image';
+import Lock from '#src/icons/Lock';
+import Tag from '#src/components/Tag/Tag';
+import { formatDurationTag } from '#src/utils/formatting';
+
+type VideoListItemProps = {
+  onClick?: () => void;
+  onHover?: () => void;
+  title: string;
+  duration: number;
+  image?: ImageData;
+  seriesId?: string;
+  seasonNumber?: string;
+  episodeNumber?: string;
+  progress?: number;
+  loading?: boolean;
+  isActive?: boolean;
+  activeLabel?: string;
+  isLocked?: boolean;
+};
+
+function VideoListItem({
+  onClick,
+  onHover,
+  title,
+  duration,
+  seriesId,
+  seasonNumber,
+  episodeNumber,
+  progress,
+  loading = false,
+  isActive = false,
+  activeLabel,
+  isLocked = true,
+  image,
+}: VideoListItemProps): JSX.Element {
+  const { t } = useTranslation('common');
+  const [imageLoaded, setImageLoaded] = useState(false);
+  const posterImageClassNames = classNames(styles.posterImage, {
+    [styles.visible]: imageLoaded,
+  });
+
+  const renderTagLabel = () => {
+    if (loading || !title) return null;
+
+    if (seriesId) {
+      return t('series');
+    } else if (seasonNumber && episodeNumber) {
+      return `S${seasonNumber}:E${episodeNumber}`;
```

Comment:
Some of this is already moved to `formatting.ts` in the latest on develop.  It would be good to reuse / add to that instead of duplicating this from Card.tsx.